std_msgs/String frame_id
std_msgs/Float32 score
std_msgs/Int32 tool_id
std_msgs/Int32 affordance_id
std_msgs/Int32 object_instance
std_msgs/Header header
geometry_msgs/Pose pose
